features/polyExplorer/src/components/clusterStories/receivingCompanies.jsx

```diff
@@ -116,20 +116,32 @@ function Industries({ entities }) {
 }
 
 export default function ReceivingCompanies({ entities }) {
+    const [selectedRecievingCompaniesTab, setSelectedRecievingCompaniesTab] =
```

Comment:
Also maybe "entities" instead of Companies, since they can be either companies or industries, right?
Although that applies to the name of the file too...